test/images/debian-testing

```diff
@@ -1 +0,0 @@
-debian-testing-f272d4495e3d91e03c2cea8c4e302e7211fd8480.qcow2
```

Comment:
This looks wrong -- why does the image link disappear completely? It should just be replaced with a new image sha?